### DISCUSSION TITLE
SQT draw ' and " for SHIFT

### DISCUSSION THREAD

**Discussion by ibeex** [caksoylar/keymap-drawer#60] [Q&A] [answered]
Hi,
how can I draw `SQT` to show ' and " for shift like is in US keyboard?

**Comment by caksoylar** [comment, marked as answer]
Hi there, are you asking how to modify the keymap YAML to show that, or how to do it during the automatic keymap YAML generation via parsing?

In the keymap YAML, you can simply change the value to add a shifted field, e.g. instead of `- "'"`, you would use `- {tap: "'", shifted: '"'}`.

If you want to do this automatically during parsing, you can create a config file (see [Customization](https://github.com/caksoylar/keymap-drawer?tab=readme-ov-file#customization)), then augment the value of `SQT` in the `qmk_keycode_map` or `zmk_keycode_map` (depending on what you are parsing) in the configuration:
```yaml
...
parse_config:
  ...
  zmk_keycode_map:
    ...
    SQT: {tap: "'", shifted: '"'}
    ...
```
You can apply the same logic for other keycodes.
